src/derive/curve.rs

```diff
@@ -49,12 +49,19 @@ macro_rules! endo {
     };
 }
 
+// These masks work for 0, 1, and 2 spare bits.
```

Comment:
The comment talks about `masks` in plural (so that's for Sign and Identity)
But the Identity mask doesn't apply to "1 spare bits" case, so this is not correct?

Maybe it should be:
```
Sign mask works for 0, 1 and 2 spare bits.
Identity mask works 0 and 2 spare bits.
```